### PR TITLE
Update composer.json

### PR DESCRIPTION
**Pull request by Arzaroth**
A straightforward change.
The goal of this PR is to have a php version requirement (I chose the lowest one from the CI setup) and also to install phpunit locally as a dev dependency.

### CODE REVIEW

**Comment by mikehaertl** [comment]
I agree to the PHP constraint but I'm not sure about phpunit. I'd rather leave this up to every developer. Some might prefer to install phpunit system wide. Actually I'm a bit confused about the `require-dev` section anyway. I think I remember that composer *always* installed everything. I may be wrong on this, though.

File: composer.json [hunk 1]
```diff
@@ -9,6 +9,12 @@
             "email": "haertl.mike@gmail.com"
         }
     ],
+    "require": {
+	"php": "^5.4.0 || ^7.0"
```
Comment (mikehaertl) [inline composer.json]: How about a single `>=5.4.0` here?

Comment (Arzaroth) [reply composer.json]: That's what I did initially, CI failed for php7. You can see why in the output of job 99.

Comment (mikehaertl) [reply composer.json]: Hmm, i see a `^5.4.0`, not a `>=5.4.0` in your initial commit.

**Comment by mikehaertl** [comment]
Regarding `require-dev`, now I remember. You explicitely have to tell composer that you *don't* want development dependencies installed with `--no-dev`. I don't like that default behaviour as it means, that you get a bloated `vendor` directory by default. So I'd say, let's leave that phpunit dependency out.

**Comment by Arzaroth** [comment]
From my point of view, it doesn't hurt to install phpunit locally even if it's already installed system wide.
I do agree about require-dev being confusing.
You have to run `composer install` with `--no-dev` switch to disable require-dev dependencies to be installed.
Although I'm not sure but I think require-dev dependencies are not installed when requiring a package that has them.

**Comment by mikehaertl** [comment]
So far the development team is very small. I can't see why we want to bloat everyone elses vendor directory with phpunit. It's not required to use this library. And the few developers here will know how to install phpunit if they need it.

So: Sorry, I'm a bit insistent on this one. :). I have a personal aversion against software that doesn't respect my valuable disk space by default.

**Comment by Arzaroth** [comment]
Duly noted, I'll leave that out of this PR :).

**Comment by mikehaertl** [comment]
Great :). Can you also give `>= 5.4.0` a try? The failed test had `^5.4.0`. I personally prefer this notation as it's slightly easier to read (I always have to look up the exact meaning of `^`).

**Pull request merged**